_includes/live-demos/premiumskinsandicons-fluent/style.css

```diff
@@ -0,0 +1,14 @@
+/* Add a border around the editor */
+.my-custom-editor-container {
+    border: 1px solid #CBCBCB;
+    border-top: 0; /* Remove top border because of the dummy header */
+}
+
+.dummy-header {
```

Comment:
This doesn't really look right? It looks right for the old `fabric` skin as it was meant to look like an old-style windows dialog, however I don't think it really makes sense for fluent. Probably a better question for @lostkeys though.

Reply:
Fluent UI is just a evolution of Fabric, and in many ways are really similar. The blue banner at the top is to show the "word-processor" use-case it's designed to be inspired by.